pallets/asset/src/lib.rs

```diff
@@ -471,6 +471,41 @@ decl_module! {
                 .map(drop)
         }
 
+        /// Utility extrinsic to batch `create_asset` and `register_custom_asset_type`.
+        #[weight = <T as Config>::WeightInfo::create_asset(
+            name.len() as u32,
+            identifiers.len() as u32,
+            funding_round.as_ref().map_or(0, |name| name.len()) as u32
+        ) + <T as Config>::WeightInfo::register_custom_asset_type(custom_asset_type.len() as u32)]
+        pub fn create_asset_with_custom_type(
```

Comment:
The new extrinsic should be added after all of the current extrinsics.  This is to make updating the Ledger app easier and to avoid an RPC problem we had with some Testnet upgrade.